guides/examples/basic-web-app.md

```diff
@@ -164,11 +260,38 @@ ockam policy create --at web --resource tcp-inlet --expression '(= subject.compo
 ockam tcp-inlet create --at /node/web --from 127.0.0.1:5432 --to /project/default/service/forward_to_db/secure/api/service/outlet
 ```
 
-Take note of the `--from` and `--to` values above. The `--from` is telling the node to listen on port 5432, the default postgres port, and to forward it `--to` the forwarder service to the database that we created in the previous section. This means requests to localhost:5432 will be forwarded to whatever node has registered as `db`, wherever it is!
+Take note of the `--from` and `--to` values above. The `--from` is telling the node to
+listen on port `5432`, the default postgres port, and to forward it `--to` the forwarder
+service to the database that we created in the previous section. This means requests to
+`localhost:5432` will be forwarded to whatever node has registered as `db`, wherever it is!
+
+Which means if you start your web app the counter will continue incrementing just as it
+did before, with zero code changes to your application
+
+You could also extend this example by moving the Postgres service into a Docker container
+or to an entirely different machine. Once the nodes are registered the demo will continue
+to work, with no application code changes and no need to expose the Postgres ports
```

Comment:
With my idea of not changing the database connection port, we could still say that there's no code change in the application if we put the port number in an env variable and had the application read from that env variable. Which is what every application does anyway, they take this information from a piece of configuration, never from the code.

Reply:
done